charts/jenkins/templates/jenkins-controller-statefulset.yaml

```diff
@@ -305,7 +305,11 @@ spec:
             - name: NAMESPACE
               value: '{{ .Values.controller.sidecars.configAutoReload.searchNamespace | default (include "jenkins.namespace" .) }}'
             - name: REQ_URL
+            {{- if .Values.controller.httpsKeyStore.enable }}
+              value: "http://localhost:{{- .Values.controller.httpsKeyStore.httpPort -}}{{- .Values.controller.jenkinsUriPrefix -}}/reload-configuration-as-code/?casc-reload-token=$(POD_NAME)"
```

Comment:
Maybe it could be useful to extract this into the helpers file and just include it here? Makes this file less `if-elsy`.

Reply:
I will give that a go!

Reply:
Done, works nicely.